src/shared/api/pluginsService/pluginsServiceKeys.ts

```diff
@@ -0,0 +1,7 @@
+import type { IGetDaoPluginsByDaoParams } from './pluginsService.api';
+
+export const pluginsServiceKeys = {
+    all: ['plugins'] as const,
+    daoPluginsByDao: (params: IGetDaoPluginsByDaoParams) =>
+        [...pluginsServiceKeys.all, 'daoPluginsByDao', params.urlParams] as const,
```

Comment:
```suggestion
export const pluginsServiceKeys = {
    allPlugins: ['plugins'] as const,
    pluginsByDao: (params: IGetDaoPluginsByDaoParams) =>
        [...pluginsServiceKeys.all, 'daoPluginsByDao', params.urlParams] as const,
```

just a naming suggestion

Reply:
changed